### ISSUE TITLE
[$10]Execute the Test case 36180: P1 Mobile - Team Calendar Shifts

### ISSUE THREAD

**Issue by ksmacleod99** [ksmacleod99/testing-project#21]
##  **Execute the Test case 36180: P1 Mobile - Team Calendar Shifts**

- Test cases are posted in Challenge Forum
- Log any issues identified to this repository and include the Test Case number and name in the Issue Description
- You may assign yourself one issue at a time. 

## **Deliverable:**

- Upon completion of the test case execution, Fill out the test case execution report and attach it with this ticket. The template is in the challenge forum.
- For each test case you are executing, attach the screenshot of your Device info screen with this ticket (If this is not attached reviewer will reject your submission)
- Provide a screen recording(video) for each test case execution (Attach the videos to this ticket itself. DON'T share it on YouTube. Share it via Google Drive, Screencast etc...)

**Devices in Scope: Please refer the Challenge Forum**

**Comment by codejamtc** [comment]
- Test case execution report: 
[Test Case Execution Report Template.xlsx](https://github.com/topcoderinc/testing-project/files/2683288/Test.Case.Execution.Report.Template.xlsx)


- Video Recording in .mp4 format
(a) Prep Steps: https://www.screencast.com/t/VxVTW1IUsJ
(b) Execution Steps:  https://www.screencast.com/t/EsHpzbBEwtt7

- Screenshot of your Device info screen: https://www.screencast.com/t/AlpHa5j92I
- Device Tested : **Samsung Galaxy Tab A 8 inch, Android 7.1.1**
- Team Name and Users you have used to execute the test case: 
(a) Team Name: **TC-codejam-Team**
(b) Users: **wipro.tester7 (Manager), wipro.tester6 (Members)**
- Test Case Result: (Pass/Fail/Blocked/Partly Executed): **Pass**
- Test Execution Time: **15min**
- Bugs:
https://github.com/topcoderinc/testing-project/issues/30

**Comment by nithyaasworld** [comment]
Thanks @codejamtc 

**Here are the Review Comments:**

1. As per step 2: _Ensure shifts are added to all members of team.
Also, add shifts to users in a group that is
unnamed. And add shifts to users from the
"ungrouped" view_

In your preparatory Video, Shifts are not added to all members of the team. Also it requires us to add Shift to a User who is in an Unnamed Group. But in your video, the user is in a named group called Normal User Group

Please make sure step2  above is done correctly and re-upload the videos.

2. As per step3: In Teams converged app: In the Schedule module,
click on the calendar icon in the top right corner.

I am not seeing the calendar icon in the top right corner. Do you see? If not, please make a note about it in your Test Case Execution Summary List's Observation column

3. Can you confirm that step 12 passed? I couldn't see that in your video

4. Whenever there is some discrepancy in the Test Case, please report it in your Test Case Execution Summary List in the Observation column noting the step number.

Ex: Step: 8 Expected value is: 

_Confirm correct day is
selected in calendar on
each day you click, it
should have a white box
around it._

What we actually have is a Blue box and not white. White box is present around the current date.
The more info you provide in the Observation column, the more we know that how thoroughly this test case execution is done.

Basically, The goal is to not miss out any bugs. 


5. As per step10: 

_Click on cancel indicators on calendar (these
should represent time off shifts). There may be
other icons depending on the type of time off_

Do you see any cancel indicators or any other icon like stated above? If not, please update in your Execution summary. If you think, its a bug, raise it. If you are not sure, Please raise it too. We will send it for review. Thanks.

**Comment by codejamtc** [comment]
@nithyaasworld Thanks for your input, it really helps. I will note down and include all the necessary info in the future challenges.

- Test case execution report: 
[Test.Case.Execution.Report.Template.xlsx](https://github.com/topcoderinc/testing-project/files/2684214/Test.Case.Execution.Report.Template.xlsx)


- Video Recording in .mp4 format
(a) Prep Steps: https://www.screencast.com/t/c1Ew1Rvc
(b) Execution Steps:  https://www.screencast.com/t/EsHpzbBEwtt7

- Screenshot of your Device info screen: https://www.screencast.com/t/AlpHa5j92I
- Device Tested : **Samsung Galaxy Tab A 8 inch, Android 7.1.1**
- Team Name and Users you have used to execute the test case: 
(a) Team Name: **TC-codejam-Team**
(b) Users: **wipro.tester7 (Manager), wipro.tester6, wipro.tester5 (Members), wipro.tester4 (Coworker)**
- Test Case Result: (Pass/Fail/Blocked/Partly Executed): **Pass**
- Test Execution Time: **15min**
- Bugs:
https://github.com/topcoderinc/testing-project/issues/30
https://github.com/topcoderinc/testing-project/issues/32
